internal/commands/scan.go

```diff
@@ -1356,6 +1356,7 @@ func runScaResolver(sourceDir, scaResolver, scaResolverParams, projectName strin
 		if err != nil {
 			return errors.Errorf("%s", err)
 		}
+		_ = os.Remove(scaFile.Name())
```

Comment:
maybe check the results of this action, and log it
something like:
err = os.Remove(scaFile.Name())
    if err != nil {
        log.Printf("Failed to remove file %s: %v", scaFile.Name(), err)
    } else {
        log.Printf("Successfully removed file %s", scaFile.Name())
    }